tests/test_check_addon.py

```diff
@@ -0,0 +1,23 @@
+import unittest
+from kodi_addon_checker import _all_repo_addons
+from kodi_addon_checker.check_addon import start
+from kodi_addon_checker.record import Record
+from kodi_addon_checker.config import Config
+
+
+class TestCheckAddon(unittest.TestCase):
+    """Integration tests for Start function present in check_addon.py"""
+
+    def setUp(self):
```

Comment:
Just `setup`, as far as I know it's one word

Reply:
I got it from https://docs.python.org/dev/library/unittest.html#unittest.TestCase.setUp